DESCRIPTION

```diff
@@ -25,23 +28,22 @@ Suggests:
     knitr,
     rmarkdown,
     survminer,
-    RColorBrewer,
-    DT,
     forcats,
     glue,
-    stringr,
-    xlsx
+    stringr
 RoxygenNote: 7.1.0
 Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
 biocViews:
 Imports: 
-    survival,
     broom,
+    RColorBrewer,
+    DT,
+    survival,
     magrittr,
     ggplot2,
     patchwork,
-    tibble,    
+    tibble,
     RBesT,
```

Comment:
Maybe something to tackle later - but we have some dependencies that maybe aren't essential. e.g. for something like`RBest` which is a bit painful, maybe we could save just the object rather than generating it for unit testing/docs.